accepted/2023/net8.0-polyfills/net8.0-polyfills.md

```diff
@@ -0,0 +1,29 @@
+# .NET 8.0 Polyfill
+
+**Owner** [Immo Landwerth](https://github.com/terrajobst)
+
+This document covers which APIs we intend to ship for older versions of .NET,
+which includes .NET Standard and .NET Framework.
+
+## Polyfills
+
+| Polyfill     | Assembly                   | Package                    | Existing? | API                    | Contacts                   |
+| ------------ | -------------------------- | -------------------------- | --------- | ---------------------- | -------------------------- |
+| TimeProvider | Microsoft.Bcl.TimeProvider | Microsoft.Bcl.TimeProvider | No        | [dotnet/runtime#36617] | [@tarekgh] [@geeknoid]     |
+| IPNetwork    | Microsoft.Bcl.IPNetwork    | Microsoft.Bcl.IPNetwork    | No        | [dotnet/runtime#79946] | [@antonfirsov] [@geeknoid] |
```

Comment:
IPNetwork seems like an over-scoped name.  When we did the SP800 compat package for crypto we found a general name (Microsoft.Bcl.Cryptography).  It seems like an equivalent here would be Microsoft.Bcl.Networking; and that any similarly needed networking types (that don't introduce a layering/dependency problem) would live in that package.

TimeProvider doesn't really have a more general bucketing that I can think of, so I don't have feedback on that line.

Reply:
We need a really strong reason to create an OOB.  For TimeProvider, it's an abstraction at the bottom of the stack that we want to plumb through types that are already OOB, e.g. Microsoft.Extensions, which also pulls it towards that path.  I don't see that for IPNetwork.  It's "just" a helper type.  What's the significant reason we need to break from our standard approach?

Reply:
Alright, looks we agreed not to do this one.